code/src/index.css

```diff
@@ -1,13 +1,47 @@
+/*** GENERAL STYLING ***/
```

Comment:
Really nice and structured CSS 👍 Good that you use different stylesheets for your components as well!